decomp2gef.py

```diff
@@ -597,6 +640,8 @@ def _handle_connect(self, args):
             self._handler_failed("not enough args")
             return
 
+        global downloaded
```

Comment:
Any time in Python you define something to be a global scope put it at the beginning of the function.